public/app/percona/settings/components/Platform/Connected/Connected.tsx

```diff
@@ -46,13 +48,21 @@ export const Connected: FC = () => {
     closeModal();
     try {
       await PlatformService.forceDisconnect();
-      appEvents.emit(AppEvents.alertSuccess, [Messages.forceDisconnectSucceeded]);
-      setDisconnecting(false);
-      dispatch(fetchServerInfoAction());
-      dispatch(fetchSettingsAction());
     } catch (e) {
-      logger.error(e);
-      setDisconnecting(false);
+      // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+      const error = e as AxiosError<ForceDisconnectErrorBody>;
+
+      if (error.response?.data?.code === 9) {
```

Comment:
Out of curiosity, what's the meaning of `9` here?

Reply:
I need to check with someone on the BE if it has a specific meaning. I might change the condition to check the error message text instead.

Reply:
@fabio-silva Just checked with Jiri, each code corresponds to a specific error message (and vice versa), so I think we should be fine here using the code to differentiate the portal disconnect error. (I'll add a comment to explain it in code)

Reply:
got it 👍